dal/user_dal.py

```diff
@@ -0,0 +1,14 @@
+from models.user import UserModel
```

Comment:
```
The name of a variable, function, or class, should answer all the big questions. It
should tell you why it exists, what it does, and how it is used. If a name requires a comment, then the name does not reveal its intent.
```
-- Clean Code page 49
same goes for files


it usually a bad practice to tell the type of a variable like PepoleList instead of Pepole 
```
Noise words are redundant. The word variable should never appear in a variable
name. The word table should never appear in a table name. How is NameString better than
Name? Would a Name ever be a floating point number? If so, it breaks an earlier rule about
disinformation. Imagine finding one class named Customer and another named
CustomerObject
```
-- Clean Code page 52

So a better name for the file will be just users / user.py in dal folder..

Reply:
Same goes for usermodel i said usermodel to show what i meant by user in my last comment but a better name will be just user